doc/tutorials/vuart_configuration.rst

```diff
@@ -108,7 +114,7 @@ connection between two VMs. The example extends the information provided in the
 
    #. For the vUART type, this example uses ``Legacy``.
 
-   #. For the virtual I/O address, this example uses ``0x2f8``.
+   #. For the virtual I/O address, this example uses ``0x3E8``.
```

Comment:
This text change says to use  ``0x3E8`` but the picture is using ``0x2E8`` and Line 143 below showing output from dmesg also is using ``0x2E8`` and talks about using ``/dev/ttyS1`` for testing the vUART communication.  These I/O addresses all need updating.